app/_includes/md/konnect/konnect-service.md

```diff
@@ -0,0 +1,22 @@
+<!-- Used in Konnect getting started guides -->
```

Comment:
Can we rename this file to `create-service.md` so that we know what it contains at a glance?